src/mlagility/api/script_api.py

```diff
@@ -189,7 +193,7 @@ def benchmark_script(
         if os.environ.get("USING_SLURM") != "TRUE":
             db.add_script(filesystem.clean_script_name(script))
 
-        for device in devices:
+        for runtime, device in zip(runtimes, devices):
```

Comment:
I'm not a big fan of `benchit bert.py --devices x86 x86 --runtimes ort torch` (ie, using the `zip` method) as a UI/UX.

What about doing all the permutations of supported devices and runtimes instead?

Reply:
That sounds like a good compromise to avoid the `--devices x86 x86 --runtimes ort torch` situation. This basically means that when multiple runtimes are used, only a single device can be selected, which I'm ok with.

Reply:
In the future it would allow `--devices nvidia x86 --runtimes ort torch torch_compiled` which would result in 6 total benchmarking runs. I intended the opposite of:

> This basically means that when multiple runtimes are used, only a single device can be selected

But in the present case, with our limited support matrix, the limitation you mention is true.

Reply:
After a chat we agreed that supporting both multiple devices and multiple runtimes on a single CLI call was confusing. We are now only allowing for a single device + one or more runtimes.